modules/userId/userId.md

```diff
@@ -148,7 +148,16 @@ pbjs.setConfig({
                 expires: 90,          // Expiration in days
                 refreshInSeconds: 8*3600 // User Id cache lifetime in seconds, defaulting to 'expires'
             },
-        }, {
+    }, {
+            name: 'nextroll',
+            params: {
+                partnerId: "1009",         // Set your real NextRoll partner ID here for production
+            },
+            storage: {
+                type: 'html5',
+                name: '_nextroll_id',
+            },
```

Comment:
since you're directly accessing localstorage rather than using the user id module to retrieve the cached id, you should remove the `storage` configuration from your examples so publishers don't use it, which could cause issues with your module.